packages/siri-sx-generator/util/siri.ts

```diff
@@ -149,6 +154,7 @@ export const getPtSituationElementFromSiteDisruption = (
                                     }
                                   : {}),
                               ...(consequence.consequenceType === "networkWide" ||
+                              (consequence.consequenceType === "journeys" && CANCELLATION_FEATURE_FLAG) ||
```

Comment:
According to the ticket, if we have a journey type consequence then we should specify the Line that is affected - we should be able to access this in the same way as we did for operators by mapping over the consequence.service.

I would check with Steven if we're implementing the LineRef changes we made in earlier tickets here i.e LineRef = lineId and PublishedLineName = lineName

Reply:
done